TESTING-INSTRUCTIONS.md

```diff
@@ -6,12 +6,12 @@
 
 1. Navigate to WooCommerce -> Home and choose "Choose payment methods".
 2. Click one of the payment methods and go through the setup process.
-3. You should be redirected to WooCommerce Home with a notice when you are done. 
+3. You should be redirected to WooCommerce Home with a notice when you are done.
 
 ### Disable the continue btn when plugins are being installed/activated #6838
 
-1. In OBW fill out store details with a USA address 
-2. Click Continue and select Fashion, apparel, and accessories, 
+1. In OBW fill out store details with a USA address
```

Comment:
My prettier is formatting markdown. We've agreed we'll use prettier to keep markdown formatted going forward so I'm leaving these changes in.